ODLA/platforms/odla_popart/custom_ops/Makefile

```diff
@@ -1,32 +1,27 @@
-CXX ? = g++ CXXFLAGS = -std =
-          c++ 14 - fPIC - g - DONNX_NAMESPACE = onnx LDLIBS =
-              -shared - lpopart - lpoplar - lpopops INCLUDES =
-                  -Iinclude - I / usr / local /
-                                  include
+CXX ?= g++
```

Comment:
can you add custom_ops.so to CMAKE build system, then use `add_odla_library()` to install custom_ops.so to $halo_build_dir/lib, ref:https://github.com/alibaba/heterogeneity-aware-lowering-and-optimization/blob/master/ODLA/platforms/CMakeLists.txt#L52